### DISCUSSION TITLE
Broken preview on GenArt

### DISCUSSION THREAD

**Discussion by vikiival** [kodadot/nft-gallery#9871] [Q&A]
Seems there is a bunch of generative art pieces that has incorrect preview or have missing metadata.

I would like to ask you if you spot a broken piece please paste it as a comment here and I will add that to the pipe.

**Comment by JustLuuuu** [comment]
**The Break:**
- https://kodadot.xyz/ahp/gallery/113-5
- https://kodadot.xyz/ahp/gallery/113-7
- https://kodadot.xyz/ahp/gallery/113-12

**Comment by vikiival** [comment]
Convergente:

- https://kodadot.xyz/ahp/gallery/112-15
- https://kodadot.xyz/ahp/gallery/112-14
- https://kodadot.xyz/ahp/gallery/112-52

**Comment by JustLuuuu** [comment]
https://kodadot.xyz/ahp/gallery/116-2404276843
https://kodadot.xyz/ahp/gallery/116-2404256490
